repository-data/application/src/main/resources/hcm-config/namespaces/hee/event.yaml

```diff
@@ -79,6 +130,24 @@ definitions:
           jcr:uuid: 7ab31fb7-1720-445c-9ca3-0ff2ce4e11ab
           hippostdpubwf:lastModificationDate: 2021-05-18T11:53:12.350+01:00
           hippostdpubwf:creationDate: 2021-05-18T11:53:12.350+01:00
+        /hipposysedit:prototype[2]:
```

Comment:
Looks like another duplicate node - could we remove if its redundant ?

Reply:
Removed